epochStart/bootstrap/epochStartBootstrapperFactory.go

```diff
@@ -14,6 +14,16 @@ func (bcf *epochStartBootstrapperFactory) CreateEpochStartBootstrapper(epochStar
 	return NewEpochStartBootstrap(epochStartBootstrapArgs)
 }
 
+// CreateStorageEpochStartBootstrapper creates a new storage epoch start bootstrapper for chain run normal
```

Comment:
Perhaps ?
```suggestion
// CreateStorageEpochStartBootstrapper creates a new storage epoch start bootstrapper for normal chain operations.
```

Also, regarding this file and `epochStart/bootstrap/sovereignEpochStartBootstrapperFactory.go`, can we please add unit tests (at least for some code coverage)?

Reply:
added tests